### ISSUE TITLE
Custom localtime to accept only YYYY-MM-DDTHH:MMZ or YYYY-MM-DDTHH:mm:SSZ

### ISSUE THREAD

**Issue by lbrossault** [json-editor/json-editor#1209]
### General information

- json-editor version: 2.8

### Expected behavior
Today, JSON document to edit include date to YYYY-MM-DDTHH:MMZ or YYYY-MM-DDTHH:mm:SSZ.
I configured JSON-Editor as below:
"Date": {
					"type": "string",
					"format": "datetime-local",
					"options": {
						"flatpickr": {"dateFormat": "Z"}
					},
					"propertyOrder": 2,
					"minLength": 1
				},

But when I edit already defined JSON file, JSON-Editor refuse defined date value and need to redefine it as YYYY-MM-DD HH:MM

### Steps to reproduce the behavior

```javascript
"Date": {
					"type": "string",
					"format": "datetime-local",
					"options": {
						"flatpickr": {"dateFormat": "Z"}
					},
					"propertyOrder": 2,
					"minLength": 1
				},
```

Any idea ?

Regards,

**Comment by a9NoDrunk** [comment]
Hi team,

Can this issue be assigned to me? This is my first time working on an issue. Could I please get some help concerning what files I should look at?

Regards,
Shawn
u6365579

**Comment by a9NoDrunk** [comment]
Hi team,

I see it is probably in the datetime.js file. Would it work to write a paragraph of the judgement code to check if the date format is correct?

Regards,
Shawn
u6365579

**Comment by schmunk42** [comment]
@a9NoDrunk Do you work with another team on this project? 

If you want to resolve an issue, please just create a PR.

**Comment by a9NoDrunk** [comment]
> @a9NoDrunk Do you work with another team on this project?
> 
> If you want to resolve an issue, please just create a PR.

Thank you for your reply!

Yes. I have a team, and my other team members are working on fixing another project issue.

And lastly, thank you for giving me this opportunity.

Regards,
Shawn

**Comment by a9NoDrunk** [comment]
Hi team,

Could you please give me each example of YYYY-MM-DDTHH:MMZ, YYYY-MM-DDTHH:mm:SSZ, and YYYY-MM-DD HH:MM? I'm not pretty sure about those dates' format.

Thanks,
Shawn

**Comment by linonetwo** [comment]
@a9NoDrunk What is the status? How is the custom date time format implemented?

**Comment by schmunk42** [comment]
I merged #1219 so have a look at https://json-editor.github.io/json-editor/datetime.html

![Bildschirmfoto vom 2023-02-22 14-48-51](https://user-images.githubusercontent.com/649031/220639039-e56ed8be-8d11-431a-a8d0-8d63a4222044.png)
